### PR TITLE
Document Record Notes

### PR DESCRIPTION
**Pull request by lokst**
Document the new Record Notes feature
- Add a new `Record Notes` article
- Update the following articles with new screenshots and field descriptions:
  - `Managing A Records`
  - `Managing AAAA Records`
  - `Managing CAA Records`
  - `Managing CNAME Records`
  - `Managing URL Records`

Belongs to https://github.com/dnsimple/dnsimple-business/issues/1586

### CODE REVIEW

File: content/articles/record-notes.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+When you update a record, you will be able to view the note that is associated with the record, if one exists.
+
+![Viewing record notes](/files/record-notes-edit.png)
+
+## Creating a record deletion note
+
+When deleting a record, you can also include a note to help you remember the reason the note was deleted.
+
+<div class="section-steps" markdown="1">
```
Comment (dallasread) [inline content/articles/record-notes.markdown:35]: This div never closes.

Comment (lokst) [reply content/articles/record-notes.markdown:35]: Good catch!

**Review by dallasread — APPROVED** [review]
Well done! We'll drop the video walkthrough into this later.

**Review by sbastn — APPROVED** [review]
Looks great! 

A couple of minor thing on the screenshots:
1. The focus is not on the notes but on the content field, so it makes the screenshot confusing 

Screenshot 2022-10-07 at 16 10 25

2. You're using a TXT record which makes for a taller form because the content field is a multiple line text input, so it takes more scrolling to get to the note fields. You can pick a different record like `A` for example. Or cut the screenshot around the note input filed with a bit of context around so it's easy to understand that it's at the bottom of the field.

3. Having more content like a google site verification code is distracting since not only you have the content edited, but also the preview area. It's a lot of info in a screenshot.

These are minor, but I think they can help understand the feature at a glance without having to stumble over other details.

**Review by itsalyse — APPROVED** [review]

**Comment by lokst** [comment]
@sbastn Thanks for the great feedback! It makes a lot of sense - I'll update the screenshots so that the focus will be on the notes field like we want them to be 🙂

**Comment by lokst** [comment]
@sbastn I've made some updates to the screenshots. Could you take a look and let me know what you think? 🙂

File: content/articles/record-notes.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+{:toc}
+
+---
+
+When creating, updating, or deleting a DNS record from the [DNS record editor](/articles/record-editor), you can include a note to explain the purpose of the record change.
+
+## Video walk-through
+
+<div class="mb4 aspect-ratio aspect-ratio--16x9 z-0">
+  <iframe src="https://www.youtube.com/embed/97jhZOmcm0?rel=0&modestbranding=1&cc_load_policy=1&cc_lang_pref=en" class="aspect-ratio--object" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```
Comment (dallasread) [inline content/articles/record-notes.markdown:22]: Maybe this time I'll get it right:

```suggestion
  <iframe src="https://www.youtube.com/embed/-97jhZOmcm0?rel=0&modestbranding=1&cc_load_policy=1&cc_lang_pref=en" class="aspect-ratio--object" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
```

**Review by sbastn — APPROVED** [review]
Thank you! Sorry for the delay in this approval

**Pull request merged**